core-primitives/top-pool-author/src/author.rs

```diff
@@ -327,20 +349,3 @@ where
 		self.top_pool.on_block_imported(hashes, block_hash)
 	}
 }
-
-impl<TopPool, TopFilter, StateFacade, ShieldingKeyRepository, OCallApi> SendState
-	for Author<TopPool, TopFilter, StateFacade, ShieldingKeyRepository, OCallApi>
-where
-	TopPool: TrustedOperationPool + Sync + Send + 'static,
-	TopFilter: Filter<Value = TrustedOperation>,
-	StateFacade: QueryShardState,
-	ShieldingKeyRepository: AccessKey,
-	<ShieldingKeyRepository as AccessKey>::KeyType: ShieldingCryptoDecrypt,
-	OCallApi: EnclaveMetricsOCallApi + Send + Sync + 'static,
-{
-	type Hash = <TopPool as TrustedOperationPool>::Hash;
-
-	fn send_state(&self, hash: Self::Hash, state_encoded: Vec<u8>) -> Result<()> {
-		self.top_pool.rpc_send_state(hash, state_encoded).map_err(|e| e.into())
```

Comment:
`SendState` could be removed, since it was only used by executed getters to send their result back through the RPC connection.